drivers/npmx/npmx_shell.c

```diff
@@ -4232,11 +4281,20 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 	SHELL_CMD(prescaler, &sub_timer_config_prescaler, "Timer prescaler selection", NULL),
 	SHELL_SUBCMD_SET_END);
 
-/* Creating subcommands (level 2 command) array for command "timer". */
-SHELL_STATIC_SUBCMD_SET_CREATE(sub_timer,
-			       SHELL_CMD(config, &sub_timer_config, "Timer config", NULL),
+/* Creating subcommands (level 3 command) array for command "timer watchdog". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_timer_watchdog,
+			       SHELL_CMD(kick, NULL, "Timer kick watchdog",
+					 cmd_timer_watchdog_kick),
 			       SHELL_SUBCMD_SET_END);
 
+/* Creating subcommands (level 2 command) array for command "timer". */
+SHELL_STATIC_SUBCMD_SET_CREATE(
+	sub_timer, SHELL_CMD(config, &sub_timer_config, "Timer config", NULL),
+	SHELL_CMD(load_config, NULL, "Timer load config", cmd_timer_load_config),
+	SHELL_CMD(start, NULL, "Timer start", cmd_timer_start),
+	SHELL_CMD(stop, NULL, "Timer stop", cmd_timer_stop),
+	SHELL_CMD(watchdog, &sub_timer_watchdog, "Timer watchdog", NULL), SHELL_SUBCMD_SET_END);
```

Comment:
What about subcommands for tasks? It will be more clear to understand.

Reply:
You mean something like that:
```
• sub_timer – timer
    ▪ sub_timer_config – config
        • sub_timer_config_compare – compare
            ▪ set
            ▪ get
        • sub_timer_config_mode – mode
            ▪ set
            ▪ get
        • sub_timer_config_prescaler – prescaler
            ▪ set
            ▪ get
    • sub_timer_task – task
        ◦ kick
        ◦ start
        ◦ stop
        ◦ strobe
```

Reply:
I think it will not be intuitive. A user does not have to know that underneath shell command task is triggered.
Moreover, wouldn't that mean that other tasks should be have `task` in command name? For example, LDSW enable/disable

Reply:
Right, it could cause applying similar tasks to the other peripherals.